typescript/ws-protocol-examples/package.json

```diff
@@ -67,7 +67,7 @@
     "@foxglove/schemas": "^1.6.2",
     "@foxglove/wasm-lz4": "^1.0.2",
     "@foxglove/wasm-zstd": "^1.0.1",
-    "@foxglove/ws-protocol": "0.7.1",
+    "@foxglove/ws-protocol": "0.7.2",
```

Comment:
is there no yarn.lock update from this? 🤔

Reply:
Nope, was also surprised.